plugins/reporters/evaluated-model/src/funTest/assets/evaluated-model-reporter-test-expected-output.yml

```diff
@@ -190,6 +221,97 @@ scan_results:
   package_verification_code: "da39a3ee5e6b4b0d3255bfef95601890afd80709"
 packages:
 - _id: 0
+  id: "Gradle:org.ossreviewtoolkit:nested-fake-project:1.0.0"
```

Comment:
Any idea why the numeric IDs of the packages seem to be swapped now? That makes the diff a bit difficult to read...

Reply:
No idea. This is always imho difficult to figure out as the IDs are assigned by Jackson.

Reply:
I've debugged this for quite a bit, and it seems to just depend on the order things get serialized. E.g. the first identifier which gets serialized gets the `0` assigned. The next identifier the `1` and so forth. So, with this change `Gradle:org.ossreviewtoolkit:nested-fake-project:1.0.0` gets serialized first. 

I believe such diffs we have to live with, as long as we use the above mentioned approach to assign identifiers.

Reply:
> E.g. the first identifier which gets serialized gets the `0` assigned.

Right, but why does the order of identifiers change with your PR, shouldn't it all be deterministic? In any case, thanks for looking into it.

Reply:
> Right, but why does the order of identifiers change with your PR, shouldn't it all be deterministic?

Hm, each time one make a small change to the test asset one needs to debug why jackson assigns the IDs in the way it does is imo raising the bar for contributions quite a bit. Can we shortcut this and live without the explanation ?

Reply:
Investigating further. Maybe @mnonnenmacher does the issue here ring any bell? (can you help?)

Reply:
I believe it is as follows:

1. The `EvaluatedModel.issues` property comes before `EvaluatedModel.packages`.
2. The issues do reference packages
3. So, the assignment of the package identifiers start when `issues` are serialized, and the ordering of
    the issues determines the IDs of the referenced packages.
4. As this commit adds a new _first_ issue, the package referenced gets the first id: `0`.

So, it seemed natural to just change property order so that `packages` are serialized before the issues, but 
that would re-create a similar issues, because there is a cyclic reference between these instances. 

@sschuberth what do you think?

Reply:
That sounds reasonable as an explanation, good enough for me, thanks!